README.md

```diff
@@ -77,7 +79,40 @@ The `Pigpiox.Clock` module provides functions that allow you to set a clock on r
 Pigpiox.Clock.hardware_clk(gpio, 2_500_000)
 ```
 
-All documentation available on [hexdocs](https://hexdocs.pm/pigpiox/).
+## Pulse-width modulation (PWM)
+
+The [`Pigpiox.Pwm` module](https://hexdocs.pm/pigpiox/Pigpiox.Pwm.html#content) provides functions that allow you to build and send waveforms with pigpiod.
+According to [Raspberry Pi's GPIO usage documentation](https://www.raspberrypi.org/documentation/usage/gpio/), here are the pins PWM is avaliable on:
+
+- Software PWM: all pins
+- Hardware PWM: GPIO12, GPIO13, GPIO18, GPIO19
+
+### Software PWM
+
+Max duty cycle is `255`.
+Here's an example of changing the brightness of an LED using software PWM.
+
+```elixir
+gpio = 12
+Pigpiox.Pwm.gpio_pwm(gpio, 255) # 100%
+Pigpiox.Pwm.gpio_pwm(gpio, 127) # 50%
+Pigpiox.Pwm.gpio_pwm(gpio, 25)  # 10%
+Pigpiox.Pwm.gpio_pwm(gpio, 2)   # 1%
+```
+
+### Hardware PWM
+
+Max duty cycle is `1_000_000`.
```

Comment:
This is fantastic! Great additions to the documentation.  I wonder if these could live on the module instead as module and function docs?

Reply:
> ... I wonder if these could live on the module instead as module and function docs?

@hez Good point. They could definitely. For now, I'd keep the change in the module minimal since other modules are kept simple.

Reply:
Oh maybe you meant max duty cycle value specifically? I'll add it in function docs.